runtime/src/main/java/org/corfudb/runtime/view/SnowflakeGuidFactory.java

```diff
@@ -0,0 +1,49 @@
+package org.corfudb.runtime.view;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Fast Globally Unique Identity (GUID) generator that returns compact ids
+ * having a notion of a universal comparable ordering.
+ *
+ *    ______________________________________________
+ *   | timestamp lower 42 bits | Node Id | Sequence |
+ *    ------------------------- --------- ----------
+ *    <------ 42 bits ---------><-10bits-><-12bits-->
+ *
+ * Created by Sundar Sridharan on 5/22/19.
+ */
+public class SnowflakeGuidFactory implements AbstractOrderedGUIDFactory {
```

Comment:
Should we really keep this name?

Its a little bit confusing, the java doc in `AbstractOrderedGUIDFactory.java` gives a reason of why we're not implementing "snowflake-style" ids, but then this is called a Snowflake factory.

Reply:
Yes, sorry for the confusion, I was in the middle of creating 2 id generators - one with just the snowflake style generation scheme and the other with corfu's token and forgot to update the comment.

Reply:
Sorry for the confusion, I started with the corfu's token based guid factory and forgot to update this comment.